rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -2294,6 +2294,17 @@ pub struct VotingPowerEconomics {
     /// Initially, set to 1/12 years.
     #[prost(uint64, optional, tag = "2")]
     pub clear_following_after_seconds: ::core::option::Option<u64>,
+
+    /// The minimum dissolve delay a neuron must have in order to be eligible to vote or
+    /// make proposals.
+    ///
+    /// Neurons with a dissolve delay lower than this threshold will not have
+    /// voting power, even if they are otherwise active.
+    ///
+    /// This value is an essential part of the staking mechanism, promoting
+    /// long-term alignment with the network's governance.
+    #[prost(uint64, optional, tag = "3")]
+    pub neuron_minimum_dissolve_delay_to_vote_seconds: ::core::option::Option<u64>,
```

Comment:
So my main question is - if we're exposing this as an API parameter, does that mean that the value is actually being used to see if a neuron can vote?  It looks like not yet.  Is there a followup PR planned to actually use the value on VotingPowerEconomics to determine neuron voting eligibility?

Reply:
This has been addressed (see [make_proposal](https://github.com/dfinity/ic/pull/4180/files#diff-98b58f03b1cbc116d2def85c2e6a81c4be8cda5e6f3cc994f0177db15d62c85eR5371) and [create_ballots_for_standard_proposal](https://github.com/dfinity/ic/pull/4180/files#diff-9f27f4156f19dc504bf0e983aeb422fac45925bf945150b8fa3eed28e25312c9R959)).